server/http/helloapp/auth.go

```diff
@@ -157,3 +157,21 @@ func (h *helloapp) DM(userID string, format string, args ...interface{}) {
 		Message:   fmt.Sprintf(format, args...),
 	})
 }
+
+func (h *helloapp) Ephemeral(userID, channelID string, format string, args ...interface{}) {
```

Comment:
You can use the AsBot function to do this, so you don't have to deal with the tokens.

Reply:
I am moving all these functions to a `client.go` file. I will send you the link when I have that finished.

Reply:
Sure sounds good :+1: